transcom/mymove
# INT- B-19713-UI-Address-PPM-Shipment-Card

## [B-19713](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A945224)

## Summary

**INTEGRATION**

Simple PR to display Origin/Destination addresses on the PPM shipment card view on customer summary page view. Origin/Destination are labeled as "From/To" in the following format in short address format(city, state, postal):  
_From: Somewhere, AL 12345 To: Somewhere, AL 12345_ 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. As customer create and submit move containing a PPM.
2. After submission verify From/To addresses are not displayed. It will be only be displayed when service counselor reviews/approves for pre-closeout and closeout. 
![Screenshot 2024-06-03 at 2 37 33 PM](https://github.com/transcom/mymove/assets/148127004/0add3879-7ccc-4f4d-ae0c-2ed581f784b6)
3. As new service counselor, review and approve for Pre-Closeout. Click <submit move details>
4. As customer, view summary page and verify "From/To" label is displayed for origin/destination addresses. **Note: old screenshot. The final product is "From/To". Not "To/From" shown in the screenshot.** 
![Screenshot 2024-06-03 at 1 05 53 PM](https://github.com/transcom/mymove/assets/148127004/174dca26-021b-446a-a6cc-ae8f81d5f11f)
5. Click <Upload PPM Documents> and upload docs and submit.
6. As counselor, review documents for PPM, APPROVE and Confirm. (Closeout)
7. As customer verify PPM shipment card view displays "From/To" with disabled <Upload PPM Documents>.
![Screenshot 2024-06-03 at 2 38 03 PM](https://github.com/transcom/mymove/assets/148127004/f1899e67-7624-462d-94b6-754efe17da69)

### Frontend
